server/selective_sync.go

```diff
@@ -31,7 +31,7 @@ func (p *Plugin) ChatMembersSpanPlatforms(members model.ChannelMembers) (bool, *
 		if isRemoteUser(user) {
 			// Synthetic users are always remote.
 			atLeastOneRemoteUser = true
-		} else if p.getAutomuteIsEnabledForUser(user.Id) {
+		} else if p.getPrimaryPlatform(user.Id) == PreferenceValuePlatformMSTeams {
```

Comment:
I change this to use the real preferences about primary platform, instead of auto-mute preference